astro/src/content/quickstarts/quickstart-android-kotlin-native.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Android Kotlin - Fusionauth SDK (Beta)
+description: Quickstart integration of a Kotlin-based Android app with FusionAuth Android SDK
+navcategory: getting-started
+prerequisites: Android Studio, Git
+section: native
+technology: Android
+language: Kotlin
+icon: /img/icons/android.svg
+faIcon: fa-android
+color: indigo
+cta: EmailListCTA
+codeRoot: https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-kotlin-android-native/main
+---
+import Intro from '/src/components/quickstarts/Intro.astro';
+import DockerSpinup from '/src/components/quickstarts/DockerSpinup.astro';
+import LoginArchitectureWeb from '/src/components/quickstarts/LoginArchitectureWeb.astro';
+import NextSteps from '/src/components/quickstarts/NextSteps.astro';
+import {RemoteCode} from '@fusionauth/astro-components';
+import RemoteContent from 'src/components/RemoteContent.astro';
+import QuickstartTshirtCTA from '/src/components/quickstarts/QuickstartTshirtCTA.astro'
+
+This quickstart is built on top of [FusionAuth Android SDK](/docs/sdks/android-sdk) demonstrating the different functionalities of the SDK available.
+
+<Intro fulltext="an Android app with Kotlin"
+    repositoryUrl="https://github.com/FusionAuth/fusionauth-quickstart-kotlin-android-native"/>
+
+## Prerequisites
+
+<RemoteContent url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-kotlin-android-native/main/README.md"
+    tags="forDocSitePrerequisites" />
+
+## General Architecture
+
+<LoginArchitectureWeb/>
+
+## Getting Started
+
+In this section, you'll get FusionAuth up and running and use `git` to create a new application.
+
+### Clone the Code
+
+First off, grab the code from the repository and change into that directory.
+
+```shell
+git clone https://github.com/FusionAuth/fusionauth-quickstart-kotlin-android-native.git
+cd fusionauth-quickstart-kotlin-android-native
+```
+
+### Run FusionAuth via Docker
+
+<DockerSpinup kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"} />
```

Comment:
The `applicationId` is incorrect as it doesn't follow the others quickstart.

Should be 
`e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`

Docker should also use 

```
  fusionauth:
    image: fusionauth/fusionauth-app:latest
```

Reply:
the kickstart is not a usual one in any case as it covers more use cases.

The image i would prefer to leave on the latest tested version.

We have a manual release process of the quickstart originating from the SDK which has a fixed version, which we have planned to automate once all other workflows are stable.

Reply:
this was a merge problem, i fixed it in all previous versions but not the latest, its all fixed now.